src/utils/opera.js

```diff
@@ -33,13 +36,18 @@ export async function isSerpSupported() {
 
       // Set `opera_serp_notification` on the `ghostery.com` domain
       // to make sure the top bar notification is not shown
-      setCookie('opera_serp_notification', 'true', 60 * 60 * 24 * 365 * 10);
+      chrome.cookies.set({
+        name: 'opera_serp_notification',
+        url: COOKIE_URL,
+        path: '/',
+        value: 'true',
+        domain: COOKIE_DOMAIN,
+        expirationDate: Date.now() / 1000 + 60 * 60 * 24 * 365 * 10,
+        httpOnly: true,
+      });
 
       isSupported = true;
     } catch {
-      // Clear out the cookie if it was set
-      setCookie('opera_serp_notification', undefined);
```

Comment:
why this was removed?

Reply:
This cannot happen - look what can cause `catch` - this code would only run if the cookie wasn't set correctly - so there is no need to clear it out.